Resources/Prototypes/Entities/Constructible/Walls/lighting.yml

```diff
@@ -11,7 +11,7 @@
     state: on
   - type: Icon
     sprite: Constructible/Lighting/light_tube.rsi
-    state: on
+    state: officon
```

Comment:
Isn't this and onicon in line 35 reversed? Since the sprite state for this is on, but the icon state is off, and viceversa for the other one.